var/spack/repos/builtin/packages/berkeleygw/package.py

```diff
@@ -182,7 +182,10 @@ def build(self, spec, prefix):
             paraflags.append("-DMPI")
 
         # We need to copy fflags in case we append to it (#34019):
+        cflags = spec.compiler_flags["cflags"][:]
         fflags = spec.compiler_flags["fflags"][:]
+        cflags.append(self.compiler.cc_pic_flag)
+        fflags.append(self.compiler.fc_pic_flag)
```

Comment:
We cannot use `flag_handler` because the build system of `berkeleygw` simply ignores `CFLAGS` and `FFLAGS`.